packages/core-components/api-report.md

```diff
@@ -1532,4 +1533,5 @@ export type WarningPanelClassKey =
 // src/components/TabbedLayout/RoutedTabs.d.ts:9:5 - (ae-forgotten-export) The symbol "SubRoute" needs to be exported by the entry point index.d.ts
 // src/components/Table/Table.d.ts:20:5 - (ae-forgotten-export) The symbol "SelectedFilters" needs to be exported by the entry point index.d.ts
 // src/layout/ErrorBoundary/ErrorBoundary.d.ts:8:5 - (ae-forgotten-export) The symbol "SlackChannel" needs to be exported by the entry point index.d.ts
+// src/layout/ProxiedSignInPage/ProxiedSignInPage.d.ts:19:5 - (ae-forgotten-export) The symbol "RefreshHeaders" needs to be exported by the entry point index.d.ts
```

Comment:
Note this warning.

But honestly, I think it could be preferable to just write the type in literally, instead of giving it the one-off name. That is, I think that `RefreshHeaders` should probably be removed, and then above we would have 

```
headers?:
  | HeadersInit
  | (() => HeadersInit)
  | (() => Promise<HeadersInit>)
```

The alternative is to export the type all the way instead, so that it appears in the api-report. But that's probably not worth it.  And if we did it should probably have a more "scoped" name such as `ProxiedSignInIdentityHeaders`

Reply:
Makes sense!